examples/otlp-exporter-node/metrics.js

```diff
@@ -4,25 +4,50 @@ const { DiagConsoleLogger, DiagLogLevel, diag } = require('@opentelemetry/api');
 const { OTLPMetricExporter } = require('@opentelemetry/exporter-metrics-otlp-http');
 // const { OTLPMetricExporter } = require('@opentelemetry/exporter-metrics-otlp-grpc');
 // const { OTLPMetricExporter } = require('@opentelemetry/exporter-metrics-otlp-proto');
-const { MeterProvider, PeriodicExportingMetricReader } = require('@opentelemetry/sdk-metrics');
+// const { ConsoleMetricExporter } = require('@opentelemetry/sdk-metrics');
+const {
+  ExponentialHistogramAggregation,
+  MeterProvider,
+  PeriodicExportingMetricReader,
+  View,
+} = require('@opentelemetry/sdk-metrics');
 const { Resource } = require('@opentelemetry/resources');
-const { SemanticResourceAttributes } = require('@opentelemetry/semantic-conventions');
+const {
+  SemanticResourceAttributes,
+} = require('@opentelemetry/semantic-conventions');
 
 // Optional and only needed to see the internal diagnostic logging (during development)
 diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.DEBUG);
 
-const metricExporter = new OTLPMetricExporter({});
+const metricExporter = new OTLPMetricExporter({
+  // headers: {
+  //   foo: 'bar'
+  // },
+});
+
+// Define view for the exponential histogram metric
+const expHistogramView = new View({
+  aggregation: new ExponentialHistogramAggregation(),
```

Comment:
@legendecas should we document this or using `Aggregation.ExponentialHistogram()` static function?

Reply:
The constructor of `ExponentialHistogramAggregation` is public and has several configurable parameters: https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/sdk-metrics/src/view/Aggregation.ts#L128 (see also https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#explicit-bucket-histogram-aggregation). So I believe this is a valid use.